pywhat/Data/regex.json

```diff
@@ -39,107 +65,107 @@
          "Cyber Security",
          "Public Key",
          "SSH Public Key"
-       ]
-    },
-    {
-       "Name": "Ethereum (ETH) Wallet Address",
-       "Regex": "(?i)^0x[a-f0-9]{40}$",
-       "plural_name": false,
-       "Description": null,
-       "Rarity": 1,
-       "URL": "https://etherscan.io/address/",
-       "Tags": [
-          "Finance",
-          "Cryptocurrency Wallet",
-          "Ethereum Wallet"
-       ]
-    },
-    {
-       "Name": "Access-Control-Allow-Header",
-       "Regex": "(?i)Access-Control-Allow: [a-z0-9\\-*]",
-       "plural_name": false,
-       "Description": "Used for [#CAE4F1][link=https://en.wikipedia.org/wiki/Cross-origin_resource_sharing]Cross-Origin Resource Sharing (CORS)[/link][/#CAE4F1]",
-       "Rarity": 1,
-       "URL": null,
-       "Tags": [
+      ]
+   },
+   {
+      "Name": "Ethereum (ETH) Wallet Address",
+      "Regex": "(?i)^0x[a-f0-9]{40}$",
+      "plural_name": false,
+      "Description": null,
+      "Rarity": 1,
+      "URL": "https://etherscan.io/address/",
+      "Tags": [
+         "Finance",
+         "Cryptocurrency Wallet",
+         "Ethereum Wallet"
+      ]
+   },
+   {
+      "Name": "Access-Control-Allow-Header",
+      "Regex": "(?i)Access-Control-Allow: [a-z0-9\\-*]",
+      "plural_name": false,
+      "Description": "Used for [#CAE4F1][link=https://en.wikipedia.org/wiki/Cross-origin_resource_sharing]Cross-Origin Resource Sharing (CORS)[/link][/#CAE4F1]",
+      "Rarity": 1,
+      "URL": null,
+      "Tags": [
          "Networking",
          "Website"
-       ]
-    },
-    {
-       "Name": "Bitcoin (₿) Wallet Address",
-       "Regex": "^[13][a-km-zA-HJ-NP-Z1-9]{25,34}$",
-       "plural_name": false,
-       "Description": null,
-       "Rarity": 1,
-       "URL": "https://www.blockchain.com/btc/address/",
-       "Tags": [
-          "Cryptocurrency",
-          "Finance",
-          "Cryptocurrency Wallet",
-          "Bitcoin Wallet"
-       ]
-    },
-    {
-       "Name": "TryHackMe Flag Format",

```

Comment:
You have changed the formatting of the whole file. This makes the diff larger than it should be and brings merge conflicts. Can your revert your formatting changes?

Reply:
Messed it up with vscode auto-formatter, fixing it now